src/uct/api/uct.h

```diff
@@ -437,6 +437,25 @@ enum {
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief   @anchor UCT_FLUSH_FLAGS Flags that control completion semantic.
+ * 
+ * These flags control completion semantic for @ref uct_ep_flush and
+ * @ref uct_iface_flush functions.
+ */
+enum {
+    UCT_FLUSH_FLAG_LOCAL   = 0,         /**< Guarantees flush of local
+                                             outstanding operations, the buffers
+                                             may be reused. */
+    UCT_FLUSH_FLAG_ACTIVE  = UCS_BIT(0) /**< All outstanding operations are
+                                             competed at least on network level
+                                             and remote peer is alive.
+                                             (currently unsupported, will be
```

Comment:
in UCT, no need to mention in the API 'currently unsupported'. Need to expose support/lack of support using iface capability bits. Also, IMHO need to phrase a better definition of 'peer is alive', also what does it mean when flushing the iface? i guess that 'all peers are alive'?